back/db/migrate/20220831142106_remove_header_bg_from_tenants.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveHeaderBgFromTenants < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :tenants, :header_bg, :string
+  end
+end
```

Comment:
I thought long and hard about removing this, particularly as the `tenant_serializer` talks recreating "... legacy serialized tenants", but in the end was satisfied that:

1. Every production tenant which had a banner image, now has a copy of that image referenced by `HomePage.first.header_bg`.
2. We either seed a header image in our dev environment (if desired), or create one based on a template (if a header image is included in the source `HomePage.first`), or manually set an image to taste when none is provided at tenant creation.

Thus, I can't see any reason that we would need to keep a copy of the homepage header image in the Tenant records. Please correct me if my thinking is wrong!

Reply:
That looks good to me.